examples/enums.rs

```diff
@@ -3,7 +3,7 @@
 use veil::Redact;
 
 #[derive(Redact)]
-#[redact(all, variant)]
+#[redact(all, variant)] // Redact all the variant names!
```

Comment:
Not something I'm really sold on, but maybe the "Redact all the variant names" comment should be changed to something like "redact variant names by default", since we don't redact the `Other` variant?